sklearn/linear_model/_glm/tests/test_glm.py

```diff
@@ -137,14 +137,14 @@ def test_glm_scalar_argument(Estimator, params, err_type, err_msg):
         glm.fit(X, y)
```

Comment:
I think this test should be completely deleted as it's redundant with the common test.

Reply:
Removed.